### DISCUSSION TITLE
Not working on android

### DISCUSSION THREAD

**Discussion by bytesagar** [gorhom/react-native-bottom-sheet#870] [Q&A] [answered]
Why is this not working on my android device. Default example provided by the repo is working on ios but in android the gesture or the animation does not show its stuck on middle of screen.
Can anyone provide me a working solution.

**Comment by pgill-rbi** [comment, marked as answer]
might have to wrap with       <GestureHandlerRootView style={{ flex: 1 }}> from react-native-gesture-handler
